tensorflow/python/kernel_tests/math_ops/bincount_op_test.py

```diff
@@ -106,7 +106,7 @@ def test_random_without_weights(self):
             np.bincount(arr, weights))
 
   @test_util.run_gpu_only
-  @test_util.disable_xla("XLA uses scatter and could be non-deterministic")
+  @test_util.disable_xla("XLA uses scatter and could be deterministic")
```

Comment:
I would phrase this "Bincount is deterministic with XLA". It uses scatter which IIRC used to be nondeterministic, but now it is.

Reply:
Can we claim this also for CPU?

https://github.com/tensorflow/tensorflow/issues/56511

Reply:
If the CPU uses a while-loop to expand scatter, than it will be slow but still deterministic. On the GPU, Scatter is expanded to a while-loop if determinism is enabled, to make it deterministic.

Reply:
Good, I've rephrased the string